website/source/docs/cluster/bootstrapping.html.md

```diff
@@ -0,0 +1,159 @@
+configuration or point Nomad client to the Nomad server service in Consul. Once
+a client establishes connection with a Nomad servers, if new servers are added
+to the cluster the addresses are propagated down to the clients along with
+heartbeat.
+
+
+### Bootstrapping a Nomad cluster without Consul
+
+At least one Nomad server's address (also known as the seed node) needs to be
+known ahead of time and a running agent can be joined to a cluster by running
+the `server-join` cli command. 
+
+For example, once a Nomad agent starts in the server mode it can be joined to an
+existing cluster with a server whose IP is known. Once the agent joins the other
+node in the cluster, it can discover the other nodes via the gossip protocol.
+
+```
+nomad server-join -retry-join 10.0.0.1
+```
+
+The `-retry-join` parameter indicates that the agent should keep trying to join
+the server even if the first attempt fails. This is essential when the other
+address is going to be eventually available after some time as nodes might take
+a variable amount of time to boot up in a cluster.
+
+On the client side, the addresses of the servers are expected to be specified
+via the client configuration.
+
+```
+client {
+    ...
+    servers = ["10.10.11.2:4648", "10.10.11.3:4648", "10.10.11.4:4648"]
+    ...
+}
+```
+
+In the above example we are specifying three servers for the clients to
+connect. If servers are added or removed, clients know about them via the
+heartbeat of a server which is alive.
+
+
+### Bootstrapping a Nomad cluster with Consul
+
+Bootstrapping a Nomad cluster is significantly easier if Consul is used along
+with Nomad. If a local Consul cluster is bootstrapped before Nomad, the
+following configuration would register the Nomad agent with Consul and look up
+the addresses of the other Nomad server addresses and join with them
+automatically.
+
+```
+{
+    "server_service_name": "nomad",
+    "server_auto_join": true,
+    "client_service_name": "nomad-client",
+    "client_auto_join": true
+}
+```
+
+With the above configuration Nomad agent is going to look up Consul for
+addresses of agents in the `nomad` service and join them automatically.  In
+addition, if the `auto-advertise` option is set Nomad is going to register the
+agents with Consul automatically too. By default, Nomad will automatically
+register the server and the client agents with Consul and try to auto-discover
+the servers if it can talk to a local Consul agent on the same server.
+
+Please refer to the (documentation)[/jobspec/servicediscovery.html] for the
+complete set of configuration options.
+
+
+### Fedarating a cluster
+
+Nomad clusters across multiple regions can be fedarated and once they are
+connected users can target the Nomad Servers in various regions from any other
+region while submitting a job or querying any Nomad API.
+
+Fedarating multiple Nomad clusters is as simple as joining a server to any other
```

Comment:
Federating multiple Nomad clusters is as simple as going servers. From a server in one region simply issue a join command to the remote region as follows: